istioctl/cmd/injector-list.go

```diff
@@ -107,7 +107,7 @@ func injectorListCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			cmd.Println()
+			fmt.Fprint(cmd.OutOrStdout(), fmt.Sprintln())
```

Comment:
Why not just using `fmt.Fprintln`?

Reply:
```suggestion
			fmt.Fprintln(cmd.OutOrStdout())
```

Reply:
![](https://user-images.githubusercontent.com/101091030/240842900-465f0a4a-877d-44d0-8e1a-b1566fad10e3.png)

I'm using OutOrStdout following the previous format, but it really doesn't seem necessary